.github/workflows/ci.yml

```diff
@@ -27,7 +27,9 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
-    - run: npm i
-    # - run: npm ci
-    - run: npm run build --if-present
-    # - run: npm test
+    - run: yarn
+    - run: yarn build
+
+    ## Docker build and push (TODO)
```

Comment:
It is recommended that this be moved into a dedicated docker.yml workflow and based on docker/build-push-action https://github.com/docker/build-push-action